doc/guide/cockpit-file.xml

```diff
@@ -8,10 +8,14 @@
   </refnamediv>
 
   <refsynopsisdiv>
+    <para>The <code>cockpit.file</code> API lets you read, write, and watch files in their entirety.  It can not
+    efficiently do random access in a big file, for example, or read streams such as <code>/dev/random</code>.
+    </para>
```

Comment:
Yes, there is no way that we can predict the file size of a file in /proc in advance, so we can't check it. But with this PR you will still be able to read those (at your own risk, as before).